src/from/parsePerkinElmer.js

```diff
@@ -32,7 +32,7 @@ export function parsePerkinElmer(text) {
   }
   return result;
 }
-
 function trim(string) {
+  // eslint-disable-next-line prefer-named-capture-group
```

Comment:
We should not disable eslint. anyway it is just a warning and the goal is to use named-capture groups (even if I don't do it)